src/integration-test/data/test-data.ts

```diff
@@ -64,6 +69,21 @@ export function createClaimData (claimantType: PartyType, defendantType: PartyTy
       type: 'submission'
     }
   }
+
+  if (claimInterestBreakdown) {
```

Comment:
Instead of having `claimInterest` and `claimInterestBreakdown` I would prefer to have `InterestType` enum, if set we either instantiate standard interest or interest breakdown.

Someone can assist you with this if you are not familiar with enums.